Articles/ShortArticles/MythOfDeveloperProductivity.md

```diff
@@ -0,0 +1,40 @@
+# Myth of Developer Productivity
+
+<!--deck text start-->
+The ability to directly effectively measure individual software developer productivity has eluded software developers for decades.  Why is this so hard and what are the alternatives? 
+<!--deck text end-->
+
+#### Contributed by [Roscoe A. Bartlett](https://github.com/bartlettroscoe), [Mark C. Miller](https://github.com/markcmiller86)
+
+#### Publication date: December, ??? 2022
+
+Here are the references I want to include: [myth-dev-prod], [cant-meas-prod], [kanban-and-scrum], [impl-lean-dev] (NOTE: We need a CC article for [impl-lean-dev].)
+
+Key points to make?
+
+* Why we care to measure productivity? => Software estimation, individual developer performance evaluation?
+
+* The number one way of improving productivity is to write less software (see [impl-lean-dev]).
```

Comment:
Hmm...carried to its extremes, this would mean we wouldn't write a single line of code and we'd be infinitely productive.

Reply:
This would go along with software reuse.